src/include/miopen/sqlite_db.hpp

```diff
@@ -230,8 +230,7 @@ class SQLiteBase
             dbInvalid = true;
             return;
         }
-
-        if(!is_system && !filename.empty())
```

Comment:
[Q] Why this is necessary?

Reply:
Not a necessary change, but the removed boolean is always true do to the previous if statement.

Reply:
[resolved]